### ISSUE TITLE
small typo?

### ISSUE THREAD

**Issue by ahmadadlan11** [adonese/visa#1]
Noebs is connected to EBS via an HTTP / RESTful apis for both consumer and merchant. EBS merchant and consumer are a wraper built on top of their TWO (**TanzWare** Online Switch System).

do you mean TranzWare ?

**Comment by adonese** [comment]
Yup!

**Comment by ahmadadlan11** [comment]
what about _EMV has built-in security features that makes it impossible for counterfeit frauds. In MagCard, we cannot guarantee a transaction originated from the_ **genunie** card-

genuine?

**Comment by adonese** [comment]
Yup it's. I wrote on sublime without spell check. Do pull request and I
will merge on a whim
